namemysprint.py

```diff
@@ -37,18 +38,39 @@
               'youthful', 'zealous', 'zen']
 
 # Cyclists, of course.
-NAMES = ['Armstrong', 'Basso', 'Boonen', 'Cancellara', 'Cavendish',
-         'Contador', 'Danielson', 'Evans', 'Froome', 'Hincapie', 'Horner',
-         'Hoy', 'Hushovd', 'Julich', 'Kloden', 'Landis', 'Leipheimer',
-         'Merckx', 'Museeuw', "O'Grady", 'Phinney', 'Powers', 'Renshaw',
-         'Schleck', 'Thomas', 'Ullrich', 'Uran', 'Valverde', 'VanGarderen',
-         'Vaughters', 'Voigt', 'Vos', 'Wiggins', 'Zabel', 'Zabriskie']
+CYCLISTS = ['Armstrong', 'Basso', 'Boonen', 'Cancellara', 'Cavendish',
+            'Contador', 'Danielson', 'Evans', 'Froome', 'Hincapie', 'Horner',
+            'Hoy', 'Hushovd', 'Julich', 'Kloden', 'Landis', 'Leipheimer',
+            'Merckx', 'Museeuw', "O'Grady", 'Phinney', 'Powers', 'Renshaw',
+            'Schleck', 'Thomas', 'Ullrich', 'Uran', 'Valverde', 'VanGarderen',
+            'Vaughters', 'Voigt', 'Vos', 'Wiggins', 'Zabel', 'Zabriskie']
 
+STARWARS = ['Ackbar', 'Amidala', 'Anakin', 'Andor', 'BB-8', 'Binks', 'C-3PO',
```

Comment:
Please alphabetize so that future contributors can easily see if a name they may want is missing.